branches-in-git/problem-set-git.md

```diff
@@ -0,0 +1,244 @@
+# Problem Set: Git
```

Comment:
Updated version that references the ada-tax-prep repo is available in https://github.com/Ada-Developers-Academy/core-interlude/tree/ar/problem-set-git